pkg/controllers/edgeconnect/config/config.go

```diff
@@ -0,0 +1,46 @@
+package config
+
+type EdgeConnect struct {
```

Comment:
I was kinda confused reading this, then I realised that it should be read like `config.EdgeConnect`, which still reads kinda weird for me, especially if it would be next to a `edgeconnect.EdgeConnect`.

I have no solution to this (other than getting used to it), but thought it is worth mentioning